cloudflare/resource_cloudflare_access_application.go

```diff
@@ -28,10 +28,9 @@ func resourceCloudflareAccessApplication() *schema.Resource {
 				Computed: true,
 			},
 			"zone_id": {
-				Deprecated: "This field will be removed in version 3 and replaced with the account_id field.",
-				Type:       schema.TypeString,
-				Optional:   true,
-				Computed:   true,
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
Should we add some schema level validation here via `ConflictsWith: []string{"other_attr"}` to make sure that we only ever have `zone_id` or `account_id`?